app/policies/vet360_policy.rb

```diff
@@ -8,4 +8,12 @@ def access?
   def military_access?
     user.edipi.present?
   end
+
+  def profile_access?
+    if Flipper.enabled?(:remove_pciu, user)
+      user.icn.present?
+    else
+      user.vet360_id.present?
+    end
+  end
 end
```

Comment:
`profile_access?` policy was created because `vet360: :access` policy only permits users when a `vet360_id` is present. 

Vet360_ID is not required for ContactInformationV2 service. The ICN will be used when the vet360_id is null.